sky/clouds/service_catalog/data_fetchers/fetch_gcp.py

```diff
@@ -485,10 +514,16 @@ def get_catalog_df(region_prefix: str) -> pd.DataFrame:
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument(
-        '--all-regions',
-        action='store_true',
-        help='Fetch all global regions, not just the U.S. ones.')
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('--all-regions',
+                       action='store_true',
+                       help='Fetch all global regions, not just the U.S. ones.')
+    group.add_argument('--zones',
+                       nargs='+',
+                       help='Fetch the list of specified zones.')
```

Comment:
This flag doesn't work because of the `region_prefix` defaulting to `us-`.

e.g. try running `python sky/clouds/service_catalog/data_fetchers/fetch_gcp.py --zones northamerica-northeast1-a`, it will error. Can you fix this?

Reply:
ops, my bad

Reply:
`python sky/clouds/service_catalog/data_fetchers/fetch_gcp.py --zones northamerica-northeast1-a` still does not work because of the `set` cast:

```
Traceback (most recent call last):
  File "fetch_gcp.py", line 536, in <module>
    EXCLUDED_REGIONS = set(args.exclude)
TypeError: 'NoneType' object is not iterable
```